Felienne/hedy
# Add language to search log results on statistics pages #1773

**Description**

Add language to the search log results on both statistics pages.

**Fix for**

#1773 

**How to test**

Unfortunately, for now log search can only be tested on Alpha.
- Log in as admin and go to /admin/stats page. Use the log search form and make sure your query yields results. Note that there is a "Lang" column included in the results table and that its value is presented correctly (e.g. you see values 'en' or 'nl').
- Log in as a teacher and go to /stats/class/$class_id. Double-check the same things about the log search results.
